src/settings.js

```diff
@@ -0,0 +1,46 @@
+const SettingsClient = require('./settingsClient')
+const Promise = require('promise')
+
+/**
+ * @class SettingsValues
+ * @classdesc
+ * <b><i> The SettingsValues class provides access to all settings values of the (most of the time authenticated) user.</i></b>
+ *
+ * @author Noveen Sachdeva
+ * @version 1.0.0
+ * @param {object}  helperFile  instance of the helpers class
+ */
+class SettingsValues {
+  constructor (helpers) {
+    this.helpers = helpers
+  }
+
+  /**
+   * Gets all settings values for the given account uuid. If no uuid is provided, settings values
+   * for the authenticated user will be fetched.
```

Comment:
also add that if no settings service is reachable, no error is returned but an empty hash.

I wonder if we should rather still return an error (like "service unavailable or so") and have the caller decide what to do, because the caller might know whether the settings service is configured or not ?

Reply:
the javascript console always shows failed requests, so the service not being available will be visible in the JS console. can't suppress that.